backend/app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::Users::SessionsController < Devise::SessionsController
+  respond_to :json
+
+  private
+  def respond_with(resource, _opts= {})
+    render json: {
+      message: 'Logged in',
+      user: current_user
+    }, status: :ok
+  end
+  def respond_to_on_destroy
+    log_out_success && return if current_user
+
+    log_out_failure
+  end
+  def log_out_success
+    render json: {message: 'You are logged out'}, status: :ok
+  end
+  def log_out_failure
+    render json: {message: 'Something went wrong'}, status: :unauthorized
```

Comment:
make the message more specific if you know its because the user isnt authorized to make the action for some reason. what are the cases where this would happen?